include/drivers/dma.h

```diff
@@ -31,7 +31,8 @@ extern "C" {
 enum dma_channel_direction {
 	MEMORY_TO_MEMORY = 0x0,
 	MEMORY_TO_PERIPHERAL,
-	PERIPHERAL_TO_MEMORY
+	PERIPHERAL_TO_MEMORY,
+	PERIPHERAL_TO_PERIPHERAL /*only supported in NXP EDMA*/
```

Comment:
If specific to NXP, some checks should be added in other vendors drivers to be sure it will not be used

Reply:
After quick check only one will not report errors IIC: `drivers/dma/dma_sam0.c`

Reply:
so need I update the dma_sam0.c driver for this? @erwango

Reply:
I'd say yes, please.

Reply:
updated